pipeline/electron/distribute-electron.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+parameters:
+    platforms: '' # electron-builder platforms, e.g. -wl for win/linux, -m for mac, etc
```

Comment:
question: can we remove the `platforms` property when we are leaving it `blank('')` or is this required for this `yaml` to be valid?

Reply:
I don't think so based on the docs: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops

Reply:
Maybe I didn't frame the question that well. My question is whether it is required for this particular `yaml` to have a property called `platforms`, to be valid? If thats the case what is the behavior it exhibits when its blank. 
Sorry if I wasn't clear first time. 👍

Reply:
If the YAML is blank then the platform defaults to match the job agent. I think this is nice behavior - for example, running it on a Mac agent will create a Mac installer.

Reply:
Awesome. Good to know.